releasenotes/acs-calling-android-sdk-release-notes.md

```diff
@@ -32,8 +32,8 @@ callTranscriptionFeature.addOnIsTranscriptionActiveChangedListener(handleCallOnI
 
 - Silence suppression for Group Calls and Teams meeting interop scenarios is enabled.
 - LocalVideoStream.switchSource(VideoDeviceInfo) will throw a `CallingCommunicationException` with `CallingCommunicationErrors.LOCAL_VIDEO_STREAM_SWITCH_SOURCE_FAILURE` error code.
-- Memory leak Renderer View for LocalVideoStream. Memory leak Renderer View for LocalVideoStream [GH#224](https://github.com/Azure/Communication/issues/224).
-- Joining a Teams call with CallOptions produces crash. [Android] App crashes when joining a microsoft teams with AudioOptions [GH#340](https://github.com/Azure/Communication/issues/340).
+- Memory leak Renderer View for LocalVideoStream [GH#224](https://github.com/Azure/Communication/issues/224).
+- Joining a Teams call with AudioOptions produces crash [GH#340](https://github.com/Azure/Communication/issues/340).
```

Comment:
Let's be specific on this one, empty AudioOptions(), ie: new JoinCallOptions().set(new AudioOptions())...